R/guide-colorbar.r

```diff
@@ -119,11 +128,18 @@ guide_colourbar <- function(
   default.unit = "line",
   reverse = FALSE,
   order = 0,
+  border = element_line(color = 'white', size=1, linetype = 'solid', lineend = 'butt'),
 
   ...) {
 
   if (!is.null(barwidth) && !is.unit(barwidth)) barwidth <- unit(barwidth, default.unit)
   if (!is.null(barheight) && !is.unit(barheight)) barheight <- unit(barheight, default.unit)
+  
+  # if logical argument supplied to tick then use defaults
+  if (identical(ticks, TRUE))
```

Comment:
Actually, thinking about it more, shouldn't the default come from the theme system?